datacenter/ucp/2.0/guides/content-trust/index.md

```diff
@@ -160,6 +160,12 @@ trusted repository.
 Valid delegation roles take the form of `targets/<delegation>`, where
 `<delegation>` does not include further slashes.
 
+You will need to add the key to at least one other delegation in order for UCP to honor the signed content:
```

Comment:
Maybe let's add "...one other delegation **in addition to `targets/releases`** ..."?